src/bpm/integration/resource_limits_test.go

```diff
@@ -104,6 +104,8 @@ var _ = Describe("resource limits", func() {
 		}
 
 		It("gets OOMed when it exceeds its memory limit", func() {
+			fmt.Fprintln(GinkgoWriter, "If this test fails then make sure you enable swap accounting! Details are in the README.")
```

Comment:
Is it worth rewording this to be: "If this test fails, then make sure you have enabled swap accounting! Details are in the README."

This wording makes it more clear that this is one potential source of failure rather than the only reason.

Reply:
Done